# Plane constraints for workspace finder
# Generated by calibrate_workspace_finder.py script
normal_x: -0.023510
normal_y: -0.826668
normal_z: -0.561685
distance: 0.574235
